src/components/ui_kit/textarea/mod.rs

```diff
@@ -51,7 +51,9 @@ pub fn TextArea<'a>(
                 },
                 onkeyup: |e| {
                     if e.data.key_code.eq(&KeyCode::Enter) && !e.data.shift_key {
-                        on_submit.call(text.to_string());
+                        if !text.trim().is_empty() {
```

Comment:
Might be better do so the following 
```suggestion
let text = text.trim();
if !text.is_empty() {
   on_submit.call(text.to_string());
}
```

(not required to accept just a suggestion)